packages/core-graphql/lib/defs/inputs.js

```diff
@@ -35,7 +34,7 @@ module.exports = `
     fee: Float
     blockId: String
     senderPublicKey: String
-    recipientId: Address
+    recipientId: String
```

Comment:
Address is the correct type. You can find it in `core-graphql/lib/resolvers/index.js`.